docs/output-targets/docs-readme.md

```diff
@@ -28,16 +33,28 @@ stencil build --docs
 ```
 
 This will cause the Stencil compiler to perform a one-time generation of README
-files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output target.
+files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output
+target.
+
+:::caution
+If you use the `--docs` flag and don't add the `docs-readme` output target to
+your Stencil configuration your documentation won't be automatically updated
+when you build your project and could get out of date.
+:::
```

Comment:
not sure this is really a `caution`, maybe it's just a `note` ?

Reply:
Yeah, I'd say "note", but I could see it going either way